### PR TITLE
docs: update spec/asyncapi.md

### PR DESCRIPTION
**Pull request by vladdoster**
---
title: "update `spec/asyncapi.md`"
---

---

**Related issue(s):**

N/A

---

- correct article agreement
- correct punctuation
- correct grammar
- reduce verbiage
- use active voice vs. passive voice to increase readability
- correct spelling

### CODE REVIEW

**Comment by derberg** [comment]
@alequetzalli hey, can I ask you for a favor? can you have a look at the proposed changes? I think they are correct and make sense but I would like expert opinion to make sure these changes do not imply different meaning of changed text 😅

File: spec/asyncapi.md [hunk 1]
```diff
@@ -144,7 +144,7 @@ By convention, the AsyncAPI Specification (A2S) file is named `asyncapi.json` or
 #### <a name="A2SObject"></a>AsyncAPI Object
 
 This is the root document object for the API specification.
-It combines resource listing and API declaration together into one document.
+It combines resource listing and API declaration into one document.
```
Comment (alequetzalli) [inline spec/asyncapi.md:147]:
```suggestion
It combines the resource listing and API declaration into one document.
```

**Review by alequetzalli — COMMENTED** [review]
@derberg Just found one tiny grammar thing and the rest LGTM 👍🏽

**Comment by derberg** [comment]
@vladdoster can you have a look at suggestion from @alequetzalli  and solve conflicts and then we can merge

**Comment by char0n** [comment]
@vladdoster are you willing to continue with this PR?

As @derberg mentioned there are some action items for this PR to proceed further:

> @vladdoster can you have a look at suggestion from @alequetzalli and solve conflicts and then we can merge

**Comment by vladdoster** [comment]
@char0n Yes, I will apply the suggested change and squash commits. Sorry for delay.

**Comment by char0n** [comment]
@vladdoster cool. Along with suggested change, conflicts needs to resolved as well.

File: spec/asyncapi.md [hunk 2]
```diff
@@ -165,17 +165,17 @@ This object can be extended with [Specification Extensions](#specificationExtens
...
+A `major`.`minor` shall be used to designate the AsyncAPI Specification version and be considered compatible with the AsyncAPI Specification specified by that `major`.`minor` version.
 The patch version will not be considered by tooling, making no distinction between `1.0.0` and `1.0.1`.
 
 In subsequent versions of the AsyncAPI Specification, care will be given such that increments of the `minor` version should not interfere with operations of tooling developed to a lower minor version. Thus a hypothetical `1.1.0` specification should be usable with tooling designed for `1.0.0`.
 
 #### <a name="A2SIdString"></a>Identifier
 
-This field represents a unique universal identifier of the [application](#definitionsApplication) the AsyncAPI document is defining. It must conform to the URI format, according to [RFC3986](https://tools.ietf.org/html/rfc3986).
+This field represents a unique universal identifier of the [application](#definitionsApplication) the AsyncAPI document defines. It must conform to the URI format, according to [RFC3986](http://tools.ietf.org/html/rfc3986).
```
Comment (dalelane) [inline spec/asyncapi.md:178]: Why change https://tools.ietf.org/html/rfc3986 to http://tools.ietf.org/html/rfc3986 ?

I know the http URL redirects to the https one anyway, so it doesn't really matter - but I still think it's better to treat the https one as canonical and link to that.

File: spec/asyncapi.md [hunk 3]
```diff
@@ -1987,15 +1987,15 @@ schemas:
 
 #### <a name="securitySchemeObject"></a>Security Scheme Object
 
-Defines a security scheme that can be used by the operations. Supported schemes are:
+Defines a security scheme that the operations can use. Supported schemes are:
 
 * User/Password.
-* API key (either as user or as password).
+* API key (either as a user or password).
```
Comment (dalelane) [inline spec/asyncapi.md:1993]: I'm not really sure what "either as a user or password" means here... that isn't specific to your change, I wasn't sure what it meant before.

**Review by dalelane — COMMENTED** [review]
Most of these look like useful improvements, thanks! I just had a couple of questions which I've added as comments.

**Comment by derberg** [comment]
@vladdoster do you want to continue with this one? there are some important questions from @dalelane

**Comment by derberg** [comment]
@vladdoster do you plan to continue with this one or shall we ask other contributors to take over?

**Comment by smoya** [comment]
Since @vladdoster does not answer anymore, I plan to close this PR and instead open an issue so someone can pick it up from where it was left. Maybe marked as a good first issue btw. 

WDYT @alequetzalli @derberg @dalelane ?

**Comment by dalelane** [comment]
> Since @vladdoster does not answer anymore, I plan to close this PR and instead open an issue so someone can pick it up from where it was left. Maybe marked as a good first issue btw.
> 
> WDYT @alequetzalli @derberg @dalelane ?

Makes sense to me 👍

**Comment by smoya** [comment]
I see you @derberg closed this PR. Are you planning to open a new issue as I suggested we could do?

**Pull request closed without merging**